Parsers/ASimDns/ProductParsers/ASimDns.yaml

```diff
@@ -15,12 +15,17 @@ References:
 Description: |
   ASIM Source Agnostic DNS Parser
 ParserName: ASimDns
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
 ParserQuery: |
+  let DisabledParser=toscalar(_GetWatchlist('ASimDisabledParser') | project Parser=column_ifexists('Parser','') | summarize make_set(Parser));
```

Comment:
Since the lists are very short (the number of disabled parsers, 10s of record worst case) I don't think it is signifcant. Also, running on several lines  "in" vs "has" is not significant. Changed it non the less.
However, the _GetWatchlist(*) function is slow, so I now added a materialize, to reduce the number of access calls.
Not sure what you mean with "lookup".